src/code42cli/cmds/devices.py

```diff
@@ -414,17 +415,14 @@ def bulk(state):
     pass
 
 
-@bulk.command(
-    name="deactivate",
-    help="""Deactivate all devices on the given list.
-            Takes as input a CSV with a 'guid' column.""",
-)
+@bulk.command(name="deactivate")
 @read_csv_arg(headers=["guid"])
 @change_device_name_option
 @purge_date_option
 @format_option
 @sdk_options()
 def bulk_deactivate(state, csv_rows, change_device_name, purge_date, format):
+    """Deactivate all devices from the provided CSV containing a 'guid' column."""
```

Comment:
If given a CSV file no rows besides a head `guid`, you get a strange error.

Reply:
I updated `read_csv()` to handle this better, in re-writing it I also discovered a few other bugs and it took me awhile to get it all right.